source/gear/swordofthesun-thiefs.js

```diff
@@ -0,0 +1,48 @@
+const { GearTemplate } = require('../classes');
+const { ESSENCE_MATCH_STAGGER_FOE } = require('../constants');
+const { getModifierCategory } = require('../modifiers/_modifierDictionary');
+const { dealDamage, changeStagger, combineModifierReceipts, removeModifier, generateModifierResultLines } = require('../util/combatantUtil');
+const { damageScalingGenerator } = require('./shared/scalings');
+
+module.exports = new GearTemplate("Thief's Sword of the Sun",
+	[
+		["use", "Removes all buffs from a foe, then deal <@{damage}> @{essence} damage to a them, plus 30 damage per unique buff removed, then gain @{bounty}g if they're downed. This is a habit, isn't it?"],
```

Comment:
I like the joke, I'm not sure if it's better to italicize it in the description put it in `.setFlavorText()`.

Reply:
I'ma go with setFlavorText(). Pillage text might be funny but I'm finding it hard to find words to abstract the gained-gold and previously-stolen-sword-itself under the same umbrella

e.g. "pillaged 30g. (This isn't the first time)" may sound misinforming when they haven't done explicitly the 30g yet in a round/game